src/App.js

```diff
@@ -1,30 +1,79 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Header from 'components/header'
 import { Container } from 'react-bootstrap'
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
 import Home from 'screens/Home'
 import About from 'screens/About'
 import Faq from 'screens/Faq'
 import ContactUs from 'screens/ContactUs'
+import Guide from 'screens/Guide'
 import { GlobalStyle, MainContainer } from 'assets/css/GlobalStyle'
+import ApiCall from 'api'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import { saveResponseData } from 'state/Response'
+
+const App = (props) => {
```

Comment:
```suggestion
const App = ({propOne, propTwo}) => {
```

Can we list out the props we expect here?

Reply:
Ok I will do it. but can you please tell me what is disadvantage with my code and what is advantage with you suggestion?

Reply:
I think its better to have the props listed, saves you from deconstructing them later in the file or using props.something, also makes it clearer what props are there just by looking at the file. Overall I guess just a popular convention I think would be useful to stick to.